projects/playground/src/app/lazy-module/lazy-module.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-lazy-module',
+  templateUrl: './lazy-module.component.html',
+  styleUrls: ['./lazy-module.component.css'],
+})
+export class LazyModuleComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit(): void {}
```

Comment:
Just a thought, what if we open some alert to signal that this was loaded? instead of having to check the web tools inspector for loaded content.

Reply:
The UI should change so that you can see it has loaded (the lazy template has some text content on it). If it's not working, it's probably down to the selector issue you mentioned above. Will test.

Reply:
Actually, the selector shouldn't break anything as it's not referenced (but I will change it anyway).